src/components/cylc/Mutation.vue

```diff
@@ -18,55 +18,57 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <template>
     <v-card class="d-inline-block pa-4">
```

Comment:
```suggestion
    <v-card>
```

There's a gap between the scrollbar and the edge of the card. We can remove this by removing this padding class (`pa-4` meaning `padding` `all sides` `4 * 4px = 16px`) from the card.

![image](https://github.com/cylc/cylc-ui/assets/61982285/31aa141d-3837-4580-8996-bd5808b3b9fd)

Also I noticed that if we remove the `inline-block` style we don't need to explicitly set the height of the `v-card-text` component